.travis.yml

```diff
@@ -43,6 +43,10 @@ install:
   - conda env create -n ale python=3.7.3
   - conda env update -f environment.yml -n ale
   - source activate ale
+  - |
+    if [ "$TRAVIS_OS_NAME" == "osx" ]; then
+      install_name_tool -change @rpath/libiomp5.dylib @loader_path/libiomp5.dylib ${CONDA_PREFIX}/lib/libmkl_intel_thread.dylib;
```

Comment:
This is a known issue with some of the numpy versions served by conda-forge. In actual use this doesn't pop up, just Travis.